lib/generators/engine_cart/install_generator.rb

```diff
@@ -52,8 +52,7 @@ def add_gemfile_include
   # the below comes from engine_cart, a gem used to test this Rails engine gem in the context of a Rails app
   file = File.expand_path("Gemfile", ENV['ENGINE_CART_DESTINATION'] || ENV['RAILS_ROOT'] || File.expand_path("../spec/internal", __FILE__))
   if File.exist?(file)
-    puts "Loading \#{file} ..." if $DEBUG # `ruby -d` or `bundle -v`
-    instance_eval File.read(file)
+    eval_gemfile file
   else
     # we get here when we haven't yet generated the testing app via engine_cart
     gem 'rails', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
```

Comment:
If you can figure out how to make that work, I agree that it would be great. I've gotten seriously confused before about what gems are available in the bundler dsl context (and how to ensure you're getting the right versions of those gems). The two cases that seem to give the most trouble are:

- brand new, clean install without engine_cart anywhere on the system (e.g. travis)
- dirty install with multiple versions of engine_cart available (e.g. local developer)